tests/FamilyHubs.ReferralUi.UnitTests/Services/WhenUsingOpenReferralOrganisationClientService.cs

```diff
@@ -49,8 +50,8 @@ public async Task ThenGetCategories()
         //Arrange
         var taxonomies = new List<TaxonomyDto>()
         {
-            new TaxonomyDto("16f3a451-e88d-4ad0-b53f-c8925d1cc9e4", "Activities, clubs and groups", "Activities, clubs and groups", null),
-            new TaxonomyDto("aafa1cc3-b984-4b10-89d5-27388c5432de", "Activities", "Activities", "16f3a451-e88d-4ad0-b53f-c8925d1cc9e4"),
+            new TaxonomyDto("16f3a451-e88d-4ad0-b53f-c8925d1cc9e4", "Activities, clubs and groups", TaxonomyType.ServiceCategory, null),
```

Comment:
this tests are not really testing anything and on top of that the method under test is not used anywhere in the project

Reply:
It is testing GetCategories() which is being used in the page LocalOfferResults